package.json

```diff
@@ -674,7 +670,7 @@
         },
         {
           "command": "powerpages.copilot.clearConversation",
-          "when": "!virtualWorkspace"
+          "when": "never"
```

Comment:
What is this change? Are we removing the option to clear conversation?

Reply:
No, just removing it from command pallete.